src/patternfly/components/Card/card.scss

```diff
@@ -31,6 +34,29 @@
     }
   }
 
+  &.pf-m-selectable {
+    position: relative;
+    cursor: pointer;
+
+    &.pf-m-selected,
+    &:hover,
+    &:focus {
+      box-shadow: var(--pf-c-card--m-selectable--hover--BoxShadow);
```

Comment:
Wonder if we should break this up into 3 variables in case it changes down the road .. i.e. they want different size shadows for the different states. Also no active state here?

Reply:
updated

Reply:
I mapped the states to `--pf-c-card--m-selectable--m-selected--BoxShadow` as it is the primary modifier for selectable cards. I would expect that changing this variable would affect the default `:hover, :focus:, :active` states5